samples/meeting-live-coding-interview/csharp/MeetingLiveCoding/ClientApp/src/components/tab.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import * as microsoftTeams from "@microsoft/teams-js";
+import { Flex, Button, Text, Card, CardBody, CardHeader } from '@fluentui/react-northstar'
+import "./tab.css"
+import { IQuestionDetails } from '../types/question';
+
+const Tab = () => {
```

Comment:
Could we rename this to Sidepanel as it would be clearer this is where that content is setup?